observability/logging/logging-6.0/log6x-about.adoc

```diff
@@ -161,7 +187,172 @@ spec:
+    mode: openshift-logging
+----
++
+[NOTE]
+====
+Ensure that the `logging-loki-s3` secret is created beforehand. The contents of this secret will vary depending on the object storage in use.
+====
+
+. Create a service account for the collector:
++
+[source,shell]
+----
+$ oc create sa collector -n openshift-logging
+----
+
+. (Optional) Create a `ClusterRole` for the collector:
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: logging-collector-logs-writer
+rules:
+- apiGroups:
+  - loki.grafana.com
+  resourceNames:
+  - logs
+  resources:
+  - application
+  - audit
+  - infrastructure
+  verbs:
+  - create
+----
++
+[NOTE]
+====
+The `ClusterRole` `logging-collector-logs-writer` is created automatically during the Cluster Logging Operator installation and does not need to be created manually.
+====
+
+. Bind the `ClusterRole` to the service account:
++
+[source,shell]
+----
+$ oc adm policy add-cluster-role-to-user logging-collector-logs-writer -z collector
+----
+
+. Install the Cluster Observability Operator.
+
+. Create a `UIPlugin` to enable the Log section in the Observe tab:
++
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1alpha1
+kind: UIPlugin
+metadata:
+  name: logging
+spec:
+  type: Logging
+  logging:
+    lokiStack:
+      name: logging-loki
+----
+
+. Add additional roles to the collector service account:
++
+[source,shell]
+----
+$ oc project openshift-logging
+$ oc adm policy add-cluster-role-to-user collect-application-logs -z collector
+$ oc adm policy add-cluster-role-to-user collect-audit-logs -z collector
+$ oc adm policy add-cluster-role-to-user collect-infrastructure-logs -z collector
+----
++
+[NOTE]
+====
+These roles enable log collection across different log types: application, audit, and infrastructure logs.
+====
+
+. Create a `ClusterLogForwarder` CR to configure log forwarding:
++
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: collector
+  namespace: openshift-logging
+  annotations:
+    observability.openshift.io/tech-preview-otlp-output: "enabled" # <1>
+spec:
+  outputs:
+    - type: lokiStack # <2>
+      name: loki-otlp
+      lokiStack:
+        target:
+          name: logging-loki
+          namespace: openshift-logging
+        dataModel: Otel # <3>
+        authentication:
+          token:
+            from: serviceAccount
+      tls:
+        ca:
+          key: service-ca.crt
+          configMapName: openshift-service-ca.crt
+  pipelines:
+   - name: my-pipeline
+     inputRefs:
+     - application
+     - infrastructure
+     outputRefs:
+     - loki-otlp
+  serviceAccount:
+    name: logger-admin
+----
+<1> Use the annotation to enable the `Otel` output, which is a Technology Preview feature.
+<2> Define the output type as `lokiStack`.
+<3> DataModel is `Otel`.
++
+[NOTE]
+====
+You cannot use `lokiStack.labelKeys` when `dataModel` is `Otel`.
+====
```

Comment:
Because we can't use the `lokistack.labelKeys` with `dataModel: Otel` anymore we should link and explain here that the `Configuring LokiStack for OTLP data ingestion` section is the new thing to accomplish the same task.